sdk/formrecognizer/azure-ai-formrecognizer/samples/v3.2/async_samples/sample_analyze_business_cards_async.py

```diff
@@ -160,4 +160,13 @@ async def main():
 
 
 if __name__ == "__main__":
-    asyncio.run(main())
+    import sys
+    from azure.core.exceptions import HttpResponseError
+    try:
+        asyncio.run(main())
+    except HttpResponseError as error:
+        filter_errors = ["Generic error", "Timeout", "Invalid request", "InvalidImage"]
```

Comment:
Are we able to filter by error code? since that's less likely to change than the message

Reply:
Added some examples using error code, but that is not always returned in the format that we handle it in HttpResponseError